### PR TITLE
Aw/ct 210

### PR DESCRIPTION
**Pull request by ash-wright123**
#### Issue:
https://mxcom.atlassian.net/browse/CT-210

#### Changes
Changed the color of the focus ring to comply with ADA standards it will now have a 3:1 contrast ratio

#### To Test: 

link this repo to the kyper-react repo and do a `npm run dev` to build kyper locally. Search for button and make sure the focus color matches the one in the token.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ---
+
 # @mxenabled/design-tokens
 
+## [1.0.3] - 01-12-2023
```
Comment (mwclemy) [inline CHANGELOG.md]: Nit: the year should be 2024 right? 😂

**Review by mwclemy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ash-wright123) [reply CHANGELOG.md]: yes

File: src/core.ts [hunk 2]
```diff
@@ -104,7 +104,7 @@ export default {
     Top: '0px -2px 8px rgba(106, 115, 129, 0.12)',
     Left: '-1px 2px 8px rgba(106, 115, 129, 0.22)',
     Right: '1px 0px 8px rgba(106, 115, 129, 0.22)',
-    Focus: '0px 0px 0px 2px rgba(82, 138, 224, 0.8)',
+    Focus: '0px 0px 0px 2px rgba(45, 100, 239, 1)',
```
Comment (mwclemy) [inline src/core.ts]: Shouldn't the opacity be 75%? I'm just reading from the attached issue.

Also if I convert `#095ADB` to rgb, I get `rgb(9, 90, 219)` and the rgba value would be `rgb(9, 90, 219, 0.75)`.

**Review by mwclemy — CHANGES_REQUESTED** [review]

File: src/core.ts (hunk 2 above)
Comment (ash-wright123) [reply src/core.ts]: Good catch, I'm not sure why it converted to (45,100,239)

**Review by mwclemy — APPROVED** [review]

**Comment by mwclemy** [comment]
LGTM! Thanks for the quick fix.

**Pull request merged**